AppConnectSwift/PasswordViewController.swift

```diff
@@ -36,19 +36,21 @@ class PasswordViewController: UIViewController, UITextFieldDelegate {
         }
         
         if PasswordViewController.validatePassword(password) {
+            userPassword = password
             self.performSegue(withIdentifier: "PasswordSuccess", sender: nil)
         } else {
             setErrorMessage("Password criteria is not met")
         }
     }
     
     override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
-        if segue.identifier == "PasswordSuccess" {
-            let securityQuestionViewController = segue.destination as! SecurityQuestionViewController
-            // Pass the userEmail, userPassword for creating account
-            securityQuestionViewController.userEmail = userEmail
-            securityQuestionViewController.userPassword = passwordField.text
+        guard segue.identifier == "PasswordSuccess", let securityQuestionViewController = segue.destination as? SecurityQuestionViewController else {
+            return
         }
+        
+        // Pass the userEmail, userPassword for creating account
+        securityQuestionViewController.userEmail = userEmail
+        securityQuestionViewController.userPassword = userPassword
```

Comment:
Could we just re-trim the password here? Either way is really fine with me, but sometimes I think adding state like this can make things more complicated, as it has to be managed / cleared / kept in-sync.